# Copyright 2023, Advanced Micro Devices, Inc
# SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note
# Downloaded from https://support.xilinx.com/s/article/000035732?language=en_US
From 92fead1e3dbcd477a4c4bdb37db43f3fc200434d Mon Sep 17 00:00:00 2001
From: Rajesh Gugulothu <rajesh.gugulothu@amd.com>
Date: Fri, 6 Oct 2023 11:53:15 +0530
Subject: [LINUX PATCH 4/5] arm64: configs: Enable
 CONFIG_VIDEO_XILINX_HDMI21RXSS for versal

Enable HDMI 2.1 Rx Subsystem driver for versal devices.

Signed-off-by: Rajesh Gugulothu <rajesh.gugulothu@amd.com>
---
 arch/arm64/configs/xilinx_versal_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/xilinx_versal_defconfig b/arch/arm64/configs/xilinx_versal_defconfig
index 1b31d7b48512..a9fe0b042675 100644
--- a/arch/arm64/configs/xilinx_versal_defconfig
+++ b/arch/arm64/configs/xilinx_versal_defconfig
@@ -135,6 +135,7 @@ CONFIG_VIDEO_XILINX_TPG=y
 CONFIG_VIDEO_XILINX_VPSS_CSC=y
 CONFIG_VIDEO_XILINX_VPSS_SCALER=y
 CONFIG_VIDEO_XILINX_DPRXSS=y
+CONFIG_VIDEO_XILINX_HDMI21RXSS=y
 CONFIG_VIDEO_XILINX_SCD=y
 CONFIG_VIDEO_XILINX_M2M=y
 # CONFIG_MEDIA_TUNER_E4000 is not set
-- 
2.17.1

